tests/sequential/opentelemetry_metrics/opentelemetry_metrics_test.go

```diff
@@ -266,6 +311,7 @@ func TestPrometheusMetrics(t *testing.T) {
 	testScalerErrors(t, data)
 	testOperatorMetrics(t, kc, data)
 	testScalableObjectMetrics(t)
+	testPauseMetric(t, data)
```

Comment:
test here following:

1. have running ScaledObject -> check that paused metric is set to false
2. pause the SO -> check that  paused metric is set to true
3. unpause the SO -> check that paused metric is set to false